src/tokentype.js

```diff
@@ -109,6 +109,7 @@ export const types = {
   starstar: new TokenType("**", {beforeExpr: true}),
 
   // Keyword token types.
+  _await: kw("await"),
```

Comment:
Isn't `await` always contextual? I don't think it should be declared a keyword like this.